### DISCUSSION TITLE
Add an overdue category for the timeboard

### DISCUSSION THREAD

**Discussion by naxey** [roovo/obsidian-card-board#175] [Ideas]
I'd love to see an overdue category or more options to define the column properties based on the due date of a task.

These are the current options:
<img width="381" alt="image" src="https://github.com/roovo/obsidian-card-board/assets/40920984/1dd05bac-6085-4ea8-9542-1b73f0e606de">

**Comment by roovo** [comment]
I will be working on this next :)

Tracking in #93
